npc/re/quests/quests_18.txt

```diff
@@ -0,0 +1,13096 @@
+	mes "The longer it goes on, the worse the impact will be.";
+	close;
+
+OnInit:
+	questinfo( QTYPE_QUEST2, QMARK_YELLOW, "isbegin_quest(16560) == 1" );
+	end;
+}
+
+// Quest: Ahab's Request. Step 3.
+wolfvill,87,109,3	script	Redcell#ep18	4_M_DESERT,{
+	if (isbegin_quest(16566) == 2) {
+		mes "[Redcell]";
+		mes "Mr. Ahab, you seem to be becoming more bearable to be around.";
+		mes "There's still a lot to watch out for, but compared to before, you're much better now.";
+		next;
+		mes "[Redcell]";
+		mes "Watching someone change is a sacred moment.";
+		mes "I should also try harder to behave like a human.";
+		close;
+	}
+	if (isbegin_quest(16562) == 1) {
+		mes "[Redcell]";
+		mes "Make that old Mr. Ahab into a person.";
+		mes "Shouldn't a person look like a person and act like a person...?";
+		next;
+		mes "[Redcell]";
+		mes "Is it too much criticism? ...Well, I'll admit it's a bit much.";
+		mes "But after working together, you'll see for yourself.";
+		close;
+	}
+	if (isbegin_quest(16561) == 1) {
+		mes "[Redcell]";
+		mes "Have you heard everything up to now?";
+		mes "You're the only one who can help us solve this suffering.";
+		next;
+		mes "[Zlan]";
+		mes "Since there's really no solution, if it doesn't work out, I'll suggest expulsion from the village by gathering opinions.";
+		next;
+		mes "[Redcell]";
+		mes "Geez, let's put that off for later...";
+		mes "But in the end, that'll probably be the last solution.";
+		next;
+		mes "[Karr]";
+		mes "It's not just us who are bothered, so if we propose it, it'll be accepted.";
+		mes "Everyone attending the village meeting is in trouble.";
+		next;
+		mes "[Redcell]";
+		mes "But! Let's leave that for later.";
+		mes "Help me a bit before the end comes. Bro.";
+		next;
+		mes "[Karr]";
+		mes "We can't communicate with that old man.";
+		mes "Why don't you go and talk to him?";
+		next;
+		mes "[Redcell]";
+		mes "Wow, that sounds great. Go and talk to him, find out why he's like that, if it's possible for him not to be like that.";
+		mes "It's a real lifetime request.";
+		next;
+		mes "[Zlan]";
+		mes "Do it as a lifetime request. I'm saving it for when I get a lover.";
+		next;
+		mes "[Redcell]";
+		mes "Yeah, that's fine. I won't be dating anyone.";
+		mes "So that's my lifelong wish. Make that old grump into a person.";
+		next;
+		mes "[Karr]";
+		mes "It seems like your wish is getting bigger.";
+		mes "Anyway, please do it. Go and talk to Mr. Ahap. ";
+		erasequest 16561;
+		setquest 16562;
+		close;
+	}
+	mes "[Redcell]";
+	mes "Someone who can't understand a word! Someone stupid! Someone who makes you say things again! Someone who shows emotions in their work!";
+	next;
+	mes "[Redcell]";
+	mes "Someone who hides their mistakes and then can't undo them! Someone who can't take responsibility for their own actions!";
+	next;
+	mes "[Redcell]";
+	mes "It's just disgusting!";
+	next;
+	mes "[Redcell]";
+	mes "Why do so many people hate you like th
```

Comment:
Since I create 1 issues regarding this, is it iRO selection for Coat is regarded as 'footwear' here ?

Reply:
Yes it's a bad translation from iRO